core/src/main/java/com/hcl/ecomm/core/config/MagentoServiceConfig.java

```diff
@@ -128,5 +129,8 @@
   @AttributeDefinition(name = "Customer Get Order Service Path", description = "This is the API path for creating order for customer", defaultValue = customerGetOrders, type = AttributeType.STRING )
   String customer_getOrder_string() default customerGetOrders;
 
+  @AttributeDefinition(name = "Pincode location", description = "This is the Dam path for pincode for state, district and city", defaultValue = {pincodeLocationPath} )
```

Comment:
@ReenaOmana : pl have a default valu in runmode as well

Reply:
@rishabh-behl  runmode has been updated. Please check